lib/bots/procbot.ts

```diff
@@ -113,6 +119,36 @@ export class ProcBot<T> {
         this.output(level, this._alertLevel, this.alertLevelStrings, message);
     }
 
+    // Process a configuration file from YAML into a nested object.
+    protected processConfiguration(configFile: string): ProcBotConfiguration | void {
+        const config: ProcBotConfiguration = yaml.safeLoad(configFile);
+
+        if (!config) {
+            return;
+        }
+
+        // Swap out known tags that become booleans.
+        const minimumVersion = ((config || {}).procbot || {}).minimum_version;
+        if (minimumVersion && process.env.npm_package_version) {
+            config.procbot.minimum_version = (process.env.npm_package_version < minimumVersion) ? false : true;
+        } else {
+            config.procbot.minimum_version = true;
+        }
```

Comment:
I'd considered that maybe a child bot could override in extenuating circumstances if there's an absolute need to do something. But maybe it does make more sense to just fail the processing and throw an error.
Will change.